help/README.md

```diff
@@ -0,0 +1,179 @@
+# Welcome to the New Help site
+
+The content of the Expensify Help is mainly composed of HTML and Markdown files (also referred as `.md` files in the documentation). We use [Jekyll](https://jekyllrb.com/) to generate a static site from those HTML and Markdown files.
+
+## HTML and Markdown basics
+
+We use the [heading markdown tags](https://www.markdownguide.org/cheat-sheet/) to create hierarchy in the article content (you can open this file in raw format and you will notice the `#` characters).
+
+In both HTML and Markdown files we use the [comment block](https://www.w3schools.com/html/html_comments.asp) to leave a note to the person who is editing the file. The content of the comment won't be rendered in the site.
+
+i.e.
+```
+<!-- This is a comment and won't be displayed in the page -->
+```
+
+# Test locally
+
+## Pre-requisites
+
+- Pull the [App repo](https://github.com/Expensify/App) in your computer.
+    - You can do it from from a Terminal command line by running `git clone https://github.com/Expensify/App.git`
+- Install jekyll in your computer by running `gem install bundler jekyll` in the command line.
```

Comment:
Oh good catch, I installed it globally but yeah makes sense to use the local version since it's already included in the Gemfile. Also I added the link to the Jekyll requirements in the Pre-requisites section.